### ISSUE TITLE
pretrain loss and AUC

### ISSUE THREAD

**Issue by ExtremelyDarkSun** [chqiwang/gpsd-rec#1]
Could you share what loss value the model converged to during pre-training on the Amazon Food dataset? Also, what rank_auc score was achieved?

**Comment by ExtremelyDarkSun** [comment]
I have observed that as the number of iterations increases, the pre-training loss will continue to decrease. When I use a sufficient number of epochs, the loss is almost zero on smaller datasets such as Amazon Beauty. Is this normal in recommendation systems (compared to LLM)? Can you share some practical tips for setting pre-training parameters to quickly converge or prevent overfitting?
